addons/core/functions/fnc_getVehicleRadios.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getVehicleRadios
+
+    Author(s):
+        Jaffa
+
+    Description:
+        Gets a list of radios in the specified vehicle
+
+    Parameters:
+        0: Object - vehicle
+
+    Returns:
+         Array of LR radio arrays in format: 
+             0 - Object - Vehicle, 1 - String - Radio Settings ID
+         or empty array if no vehicle radios found
+
+    Example:
+        _radios = _vehicle call TFAR_fnc_getVehicleRadios;
+*/
+if !(params [["_vehicle",objNull,[objNull]]]) exitWith { ERROR_1("TFAR: Vehicle must be passed to getVehicleRadios. %1 was passed instead.", _vehicle) };
```

Comment:
You could return an empty array even if the wrong input was given.

Reply:
There's no use case in which a non-object would need to be passed into the function.